File: ePrijevozSarajevo/ePrijevozSarajevo.Model/Request.cs
namespace ePrijevozSarajevo.Model
{
    public class Request
    {
        public int RequestId { get; set; }
        public int UserStatusId { get; set; }
        public Status? UserStatus { get; set; }
        public User? User { get; set; } = null;
        public int UserId { get; set; }
        public DateTime? DateCreated { get; set; }
        public bool Approved { get; set; } = false;
        public string? RejectionReason { get; set; }
        public bool? Active { get; set; } = true;
    }
}

File: ePrijevozSarajevo/UI/eprijevoz_mobile/lib/models/request.dart
import 'package:json_annotation/json_annotation.dart';
part 'request.g.dart';

@JsonSerializable()
class Request {
  int? requestId;
  int? userStatusId;
  int? userId;
  DateTime? dateCreated;
  bool? approved;
  String? rejectionReason;
  bool? active;

  Request(
      {this.requestId,
      this.userStatusId,
      this.userId,
      this.dateCreated,
      this.approved,
      this.rejectionReason,
      this.active});

  factory Request.fromJson(Map<String, dynamic> json) =>
      _$RequestFromJson(json);

  Map<String, dynamic> toJson() => _$RequestToJson(this);
}
